samples/react-query/hook-mutator/orval.config.ts

```diff
@@ -30,6 +31,14 @@ export default defineConfig({
         //     },
         //   },
         // },
+        // for ES Module imports
```

Comment:
@henkerik 
The commented out tests are not necessary because it is unclear what purpose they serve in the future. Please remove them.